src/lib/Components/Bidding/types.d.ts

```diff
@@ -0,0 +1,60 @@
+export interface BidderPositionResult {
```

Comment:
#minor but can we alphabetize these?

Also I imagine that `Address` will eventually want to be extracted further, but for now 👍

Reply:
Ideally we shouldn't have to define and maintain the types for data coming from MP (we have[ `BidderPositionResult` for this call](https://github.com/artsy/emission/blob/6cf7031c7a880e13ec1b04f64c1cc8a84f4589e6/src/lib/Components/Bidding/Screens/ConfirmBid.tsx#L71-L82)). I'd prefer to organize `PaymentCardTextFieldParams` and `StripeToken` next to each other since these two are both Stripe-related, but for now that's :+1